docs/user-guide/ide-progress-status.md

```diff
@@ -0,0 +1,156 @@
+---
+description: Check if your favourite language is supported.
+---
+
+# Lite XL IDE Progress Status
+
+intro
+
+## Development boards
+
+- [Intellisense](https://github.com/users/PerilousBooklet/projects/6/views/1)
+- [Utilities](https://github.com/users/PerilousBooklet/projects/7/views/1)
+- [Build Integration](https://github.com/users/PerilousBooklet/projects/8/views/1)
+- [Debug Integration](https://github.com/users/PerilousBooklet/projects/9/views/1)
+- [Profiling Integration](https://github.com/users/PerilousBooklet/projects/10/views/1)
+
+## Features
+
+- Syntax highlighting for 100+ languages
+- Intellisense support for 25+ languages
+- Language-specific treeview icons
+- VSCode-like minimap
+- Project-wide text and filename search
+- Multi-cursor editing
+- Single/multi-line commenting with shortcuts
+- Go to line n command
+- Todo treeview
+- Tooltips
+- Integrated terminal
+- Git integration
+- Builder integration
+- Debugger integration
+- Markdown rendering support via [ghmarkdown](https://github.com/lite-xl/lite-xl-plugins/blob/master/plugins/ghmarkdown.lua) or [mdpreview](https://github.com/Not-a-web-Developer/lite-xl-mdpreview)
+
+### WIP
+- Foldable code blocks
+- Horizontal scrolling for project treeview
+- Ligatures support
+- Github Copilot integration
+- Project template manager
+- Project-wide refactoring
+- External libraries item in project treeview
+- Custom Canvas API to allow an external renderer to draw on a surface that Lite XL will render on screen
+
+### TODO
+- Document symbols treeview
+- Simultaneous tag rename for all languages that require it
+
+## Languages
+
+<!-- check: <span>&#9989;</span> or 1004 -->
+<!-- x: <span>&#x274c;</span> or 1008 -->
+
+| Language | Syntax High. | LSP support | Linting | Snippets | Formatter | Builder | Debugger |
+|------------------|:--------------:|:-------------:|:---------:|:----------:|:----------:|:-----------:|:---------:|
+| Arduino | <span>&#9989;</span> |  |  |  |  |  |  |
```

Comment:
Hi, just a heads up, you can probably use the shortcode for the emojis.
You can find them in this website.
https://emojipedia.org/check-mark#technical

Reply:
Thanks, I'll take a look.